runtime/doc/usr_04.txt

```diff
@@ -349,9 +350,8 @@ Notice that "yw" includes the white space after a word.  If you don't want
 this, use "ye".
 
 The "yy" command yanks a whole line, just like "dd" deletes a whole line.
-Unexpectedly, while "D" deletes from the cursor to the end of the line, "Y"
-works like "yy", it yanks the whole line.  Watch out for this inconsistency!
-Use "y$" to yank to the end of the line.
+Similarly, "D" deletes from the cursor to the end of the line, and "Y" yanks
```

Comment:
Do we even need this line anymore? I thought the main point of the prior line was to indicate that the behavior was unexpected, but now that the behavior is consistent we can probably drop line 353 and 354

Reply:
That's a good point. Thanks.